src/hooks/__tests__/useRecoveryTxState.test.tsx

```diff
@@ -15,10 +15,10 @@ describe('useRecoveryTxState', () => {
       jest.setSystemTime(0)
 
       const delayModifierAddress1 = faker.finance.ethereumAddress()
-      const nextTxHash1 = faker.string.hexadecimal()
+      const nextTxHash1 = faker.string.hexadecimal({ length: 10 })
```

Comment:
I increased these as the possibility of duplicate faked data was relatively high and broke tests.